doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -419,19 +425,21 @@ Admin REST API provides the following actions:
 * [Get server-side endpoint profile schemas]({{root_url}}Programming-guide/Server-REST-APIs/#!/Profiling/getServerProfileSchemasByApplicationToken)
 * [Update server-side endpoint profile]({{root_url}}Programming-guide/Server-REST-APIs/#!/Profiling/updateServerProfile)
```

Comment:
How does, for example, "Delete endpoint" call relate to the server-side profile? Copy-paste?